src/tests/fixtures/test.js

```diff
@@ -66,3 +66,17 @@
   "turbo:frame-missing",
   "turbo:reload"
 ])
+
+window.customElements.define('custom-link-element', class extends HTMLElement {
+  constructor() {
+    super()
+    this.attachShadow({ mode: 'open' })
+  }
+  connectedCallback() {
+    this.shadowRoot.innerHTML = `
+      <a href="${this.getAttribute('link')}">
+        <slot></slot>
```

Comment:
This is minor but I think using `slot` is more common than using a separate attribute.

Reply:
This might actually change the subject that is being tested, so I have changed this back to the original code.